DNN Platform/DotNetNuke.Web.Razor/DotNetNukeWebPage.cs

```diff
@@ -59,6 +60,14 @@ public abstract class DotNetNukeWebPage<TModel> :DotNetNukeWebPage where TModel
     {
         private TModel _model;
 
+        public DotNetNukeWebPage()
+        {
+            var model = Globals.DependencyProvider.GetService<TModel>();
+            Model = model == null ?
+                Activator.CreateInstance<TModel>() :
+                model;
```

Comment:
This could be simplified to Model = model ?? Activator.CreateInstance<TModel>();

Reply:
Yes, we should be able to simplify this. Thanks for the suggestion